src/App.tsx

```diff
@@ -1,27 +1,35 @@
-import * as React from 'react';
 import { Menu } from './navigation';
 import { useLocation } from 'react-router-dom';
+import ErrorPage from './Error';
+import VetList from './components/VetList';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Root from './Root';
+import OwnerList from './components/OwnerList';
```

Comment:
You can optimize imports, by keeping `OwnerList` and `VetList` exported from one path, you can add index.ts file under `/components`. 

So the import expression becomes: ``import {OwnerList, VetList} from './components';``